MAINTAINERS.md

```diff
@@ -1,17 +1,91 @@
 # Maintainer Hints
 
-Review the [next release](
+## Creating a New Release
+
+These are the steps for a Release Manager to create and publish a
+new release of the Agoric SDK. This combines the process of
+GitHub-based release managment and publication together with NPM-based
+publication of the SDK and its individual packages.
+
+### Prerequisites
+
+#### NPM Account
+
+Sign up for an [NPM account](https://www.npmjs.com/signup) with
+some form of 2FA. Then request that an administrator add your NPM
+user ID to the `@agoric` organization and the `agoric` package.
+
```

Comment:
Did you want to mention the need for `node`, the `npm install --global yarn`, and (maybe? to at least test that the go code compiles?) a golang install of the correct version?

Reply:
Go is pretty good at backward-compatibility, so I added instructions to follow the "Getting Started" guide for `node`, `yarn`, and `git`, plus the latest Go stuff. Note - our upgrade-8 instructions under "Syncing Up on Mainnet" omit the Go build instructions. I'll inform Ari and Molina.